numcodecs/tests/test_pickles.py

```diff
@@ -54,5 +55,7 @@ def test_repr():
     check_repr("Pickle(protocol=-1)")
 
 
+@pytest.mark.skipif(sys.byteorder != 'little',
+                    reason='Pickle does not restore byte orders')
```

Comment:
Is there any more information about why this is?

Reply:
No, not much specifically; all I can say is this: https://github.com/zarr-developers/numcodecs/issues/181#issuecomment-477507464